### ISSUE TITLE
Re-Implement navigationDots for ProductSlider

### ISSUE THREAD

**Issue by marcelbrode** [shopware/shopware#7795]
### PHP Version

-

### Shopware Version

6.7-RC1

### Affected area / extension

Platform(Default)

### Actual behaviour

Navigation Dots affect the styling, while it's not even possible to configure dots at all

### Expected behaviour

Navigation Dots should be configurable, so that every state is bug free

### How to reproduce

Use the ProductSlider in CMS and configure navigation Arrows -> Dots will also change styling
